### PR TITLE
Add ability to disable accounts

### PR DESCRIPTION
**Pull request by MatthewPhinney**
Needs to be merged in conjunction with https://github.com/interledger/five-bells-ledger/pull/90.

### CODE REVIEW

File: schemas/Account.json [hunk 1]
```diff
@@ -31,6 +31,10 @@
     "is_admin": {
       "description": "admin flag",
       "type": "boolean"
+    },
+    "disabled": {
+      "description": "Account is disabled",
```
Comment (clark800) [inline schemas/Account.json]: Is this permanent or reversible? We should clarify in the description.

Comment (MatthewPhinney) [inline schemas/Account.json]: It is reversible. The idea is to allow admins to disable or re-enable accounts at any time. I will update the description.

File: schemas/Account.json [hunk 2]
```diff
@@ -31,6 +31,10 @@
     "is_admin": {
       "description": "admin flag",
       "type": "boolean"
+    },
+    "disabled": {
+      "description": "Admin users may temporarily disable an account by setting this flag",
```
Comment (clark800) [inline schemas/Account.json]: "temporarily" makes it sound like it *cant* be permanent, but this is potentially permanent, but reversible right?

Comment (MatthewPhinney) [inline schemas/Account.json]: It is permanent, until an admin decides to reverse it. Maybe "Admin users may disabled/enable an account with this flag." or something along those lines?

Comment (clark800) [inline schemas/Account.json]: That works, with "disabled" -> "disable".

Comment (MatthewPhinney) [inline schemas/Account.json]: What if we call the field `is_disabled` for consistency with the `is_admin` field?
I'm inclined to use the past tense for this field for consistency across put/get/db column names (when a user GETs an account or looks in the db, they want to see the `is_disabled` flag, not the `disable` flag).

Comment (clark800) [inline schemas/Account.json]: I meant in your quotation there is a typo. `disabled` is sufficiently clear, but you're right that `is_disabled` is more consistent with `is_admin`. I guess we should either go with `is_disabled` or rename `is_admin` to `admin`.

**Comment by clark800** [comment]
LGTM

**Comment by justmoon** [comment]
LGTM

**Pull request merged**